packages/angular/src/generators/mfe-remote/schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "$id": "NxMFERemote",
+  "cli": "nx",
+  "title": "Nx MFE Remote App",
+  "description": "Create an Angular MFE Remote App",
```

Comment:
nit: update the descriptions in this file to use full words so the docs read better